### PR TITLE
Deployment history - Fixed blank screen

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Getting failed to load the view after clicking on the browser's back button on the deployment history page & clicking on deployment history

Fixes - https://github.com/devtron-labs/devtron/issues/2176

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

##To Reproduce

1. Go to AppDetails page
2. Navigate to the Deployment history tab
3. Click on the browser's back button
4. Observe

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/cdDetails/CDDetails.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default function CDDetails() {
 
     const { path } = useRouteMatch()
     const { pathname } = useLocation()
-    const { push } = useHistory()
+    const { push, replace } = useHistory()
```
Comment (shivani170) [inline src/components/app/details/cdDetails/CDDetails.tsx:81]: Can remove push if not using anywhere

File: src/components/app/details/cdDetails/CDDetails.tsx [hunk 2]
```diff
@@ -709,19 +708,6 @@ const SelectEnvironment: React.FC<{ environments: AppEnvironment[] }> = ({ envir
         }
     }
```
Comment (shivani170) [inline src/components/app/details/cdDetails/CDDetails.tsx:710]: Hope this deletion does not make any other impact. Please do confirm

File: src/components/app/details/cdDetails/CDDetails.tsx (hunk 1 above)
Comment (milindk8) [reply src/components/app/details/cdDetails/CDDetails.tsx]: Done

**Review by vivek-devtron — APPROVED** [review]

File: src/components/app/details/cdDetails/CDDetails.tsx (hunk 2 above)
Comment (milindk8) [reply src/components/app/details/cdDetails/CDDetails.tsx:710]: no impact

**Review by shivani170 — APPROVED** [review]

**Pull request merged**